kserve/README.md

```diff
@@ -0,0 +1,121 @@
+# KServe
+
+KServe comes with two component:
+
+1. [KServe](#KServe)
+2. [KServe runtimes](#KServe-Runtimes)
```

Comment:
is there a reason to separate this part(kserve runtime) from `kserve`?

Reply:
KServe does that, I just followed that. I think it is because the runtimes are cluster-scoped and kustomize, unnecessarility, adds the namespace to all the resources. So. `kserve/base/runtimes/kustomization.yaml` does not define the namespace field.

Reply:
OpenDataHub does not support cluster-scoped runtime. Do you know how to create namespace-scoped runtime(ServingRuntime)?  Plus, OpenDataHub modelserving does not create any runtime because the ServingRuntime will be created by ODH console using [this template](https://github.com/opendatahub-io/odh-dashboard/blob/main/manifests/modelserving/ovms-ootb.yaml).  At the moment, we only support ovms.

Reply:
Ah, KServe also defines [ServingRuntime](https://kserve.github.io/website/0.10/modelserving/servingruntimes/) instead of the cluster scoped one. So we could omit it here completely and deploy the `ServingRuntime` from the template then.

Reply:
In order to finalize the discussion my understanding is that we dont use any runtimes as they are clusterscoped and not supported.